src/docs/Icons/Icons.tsx

```diff
@@ -46,7 +46,7 @@ const iconStyles = {
 
 const getStyledIcon = (icon: string) => {
   const iconName = `Icon${icon}`;
-  const Icon = allIcons[iconName as keyof typeof allIcons];
+  const Icon: any = allIcons[iconName as keyof typeof allIcons];
```

Comment:
This basically used to be `any` in the past as well, since the `Icon` variable did not have any types inferred from the suomifi-icons library